data/transition-sites/dclg_firekills.yml

```diff
@@ -0,0 +1,7 @@
+---
+site: dclg_firekills
```

Comment:
This site can probably be folded into https://github.com/alphagov/transition-config/blob/master/data/transition-sites/directgov_firekills.yml